packages/dataviews/src/components/dataform/stories/index.story.tsx

```diff
@@ -1272,3 +1415,23 @@ export const Validation = {
 export const Visibility = {
 	render: VisibilityComponent,
 };
+
+export const MultipleSummaryFields = {
```

Comment:
Can we integrate this in the existing `LayoutPanel` story?

Reply:
(It doesn't need to be everything, as long as there's one or two examples for each it's fine (a single summary field + a list of summary fields).